skmultilearn/adapt/mlknn.py

```diff
@@ -188,7 +188,7 @@ def _compute_cond(self, X, y):
             for label in range(self._num_labels):
```

Comment:
1. make sure the definition of `deltas` still works with nans
2. make sure label with all nans still works
3. figure out how to fork repo / use this version in our code

Reply:
1. `deltas` represents the label-wise sum of 1's for the neighbors of an instance. I think this will work with nans. I can discuss this more tomorrow.
2. todo
3. dry run - should work. I think this is a rare case tbf
4. lil should work with nans, although I recommend using dense arrays wherever numpy apis like isfinite or nansum are needed
Can discuss further tomorrow, and discuss plan/experiments to test this approach

Reply:
re 2, check how genAI uses colbert maybe